docs/guides/qpu-information.ipynb

```diff
@@ -0,0 +1,805 @@
+       "  datetime.datetime(2025, 11, 25, 6, 52, 23, tzinfo=tzlocal())),\n",
+       " 'T2': (4.569656984603511e-05,\n",
+       "  datetime.datetime(2025, 11, 25, 6, 53, 9, tzinfo=tzlocal())),\n",
+       " 'readout_error': (0.01416015625,\n",
+       "  datetime.datetime(2025, 11, 25, 11, 42, 11, tzinfo=tzlocal())),\n",
+       " 'prob_meas0_prep1': (0.02734375,\n",
+       "  datetime.datetime(2025, 11, 25, 11, 42, 11, tzinfo=tzlocal())),\n",
+       " 'prob_meas1_prep0': (0.0009765625,\n",
+       "  datetime.datetime(2025, 11, 25, 11, 42, 11, tzinfo=tzlocal())),\n",
+       " 'readout_length': (1.56e-06,\n",
+       "  datetime.datetime(2025, 11, 25, 11, 42, 11, tzinfo=tzlocal()))}"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# calibration data with detailed properties of qubit 0\n",
+    "backend.properties().qubit_property(0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "1379efb5-359a-4246-9603-fa5d007f9f8a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Qubit 126 Properties:\n",
+      "  T1: 0.00011004519582612597 seconds\n",
+      "  T2: 6.519695327562977e-05 seconds\n",
+      "  Frequency: None Hz\n",
+      "  Anharmonicity: None Hz\n",
+      "  Readout Error: 0.003662109375\n",
+      "  P(0 | 1): 0.0048828125\n",
+      "  P(1 | 0): 0.00244140625\n",
+      "  Readout Length: 1.56e-06 seconds\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Retrieve qubit properties\n",
+    "qubit_index = 126  # Replace with your qubit index\n",
+    "qubit_props = backend.properties().qubit_property(qubit_index)\n",
+    "\n",
+    "# Access specific properties\n",
+    "t1 = qubit_props.get(\"T1\", (None,))[0]\n",
+    "t2 = qubit_props.get(\"T2\", (None,))[0]\n",
+    "frequency = qubit_props.get(\"frequency\", (None,))[0]\n",
+    "anharmonicity = qubit_props.get(\"anharmonicity\", (None,))[0]\n",
+    "readout_error = qubit_props.get(\"readout_error\", (None,))[0]\n",
+    "prob_meas0_prep1 = qubit_props.get(\"prob_meas0_prep1\", (None,))[0]\n",
+    "prob_meas1_prep0 = qubit_props.get(\"prob_meas1_prep0\", (None,))[0]\n",
+    "readout_length = qubit_props.get(\"readout_length\", (None,))[0]\n",
+    "\n",
+    "print(f\"Qubit {qubit_index} Properties:\")\n",
+    "print(f\"  T1: {t1} seconds\")\n",
+    "print(f\"  T2: {t2} seconds\")\n",
+    "print(f\"  Frequency: {frequency} Hz\")\n",
+    "print(f\"  Anharmonicity: {anharmonicity} Hz\")\n",
+    "print(f\"  Readout Error: {readout_error}\")\n",
+    "print(f\"  P(0 | 1): {prob_meas0_prep1}\")\n",
+    "print(f\"  P(1 | 0): {prob_meas1_prep0}\")\n",
+    "print(f\"  Readout Length: {readout_length} seconds\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "67525443-070e-4798-99b3-72a599bf96c8",
+   "metadata": {},
+   "source": [
+    "### Instruction properties\n",
```

Comment:
I know you're not done merging the pages, but instruction properties are also available on IQP - e.g. single qubit gate length under `Qubit`. 

Similarly, everything available on IQP should be queryable from Qiskit.